### PR TITLE
Awesome class

### PR DESCRIPTION
**Pull request by ghermaico135**
Adding and removing of books form awesome books using class

### CODE REVIEW

File: awesomeBook.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+<button type="submit" class="btn btn-danger removeBtn" id="${book.id}">
+Remove
+</button>
+</div>
+</div>
+    `;
+    });
+
+    return content;
+  }
```
Comment (AmaduKamara) [inline awesomeBook.js:51]: - [ ] Good job on the add and remove functionalities, adding a book and removing one is working fine as expected but when the page is reloaded, no book is shown on the page. Kindly work on showing the books even when the page is refreshed 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @ghermaico135, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍🏽
✅ The Linter checks passed successfully 👍🏽
✅ Used JavaScript classes well 👍🏽
✅ Add and remove features work fine as expected 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] When the page is refreshed, all books disappear from the page. Kindly work on that so the books remain visible on the page even after the page is refreshed 👍🏽

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by ghermaico135** [comment]
Fixing visibility of books when loading and refreshing

**Review by osamaashraf6 — APPROVED** [review]
Hi @ghermaico135 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF

## Highlights🎉🎉

- [x] No linter errors present👌
- [x] The PR has a proper title and a descriptive body 👍
- [x] Used JavaScript classes well 💯

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**